internal/services/server.go

```diff
@@ -54,9 +55,12 @@ func RegisterGrpcServices(
 	v1alpha1.RegisterSchemaServiceServer(srv, v1alpha1svc.NewSchemaServer(prefixRequired))
 	healthManager.RegisterReportedService(v1alpha1.SchemaService_ServiceDesc.ServiceName)
 
-	v1.RegisterPermissionsServiceServer(srv, v1svc.NewPermissionsServer(dispatch, maxDepth))
+	v1.RegisterPermissionsServiceServer(srv, v1svc.NewPermissionsServer(dispatch, maxDepth)) //TODO Register lookupwatch
 	healthManager.RegisterReportedService(v1.PermissionsService_ServiceDesc.ServiceName)
 
+	v1lookupwatch.RegisterLookupWatchServiceServer(srv, v1svc.NewLookupWatchServer(dispatch, maxDepth))
```

Comment:
For now, this should be put behind a CLI flag and defaulted to `false`, since the long-term location for it still remains undecided

Reply:
done